main.py

```diff
@@ -1,10 +1,46 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
```

Comment:
Ensure that the 'players.json' file exists and is in the correct format. Consider adding error handling to manage cases where the file might not be found or the JSON is malformed.